src/components/NavItem.svelte

```diff
@@ -11,7 +11,7 @@
   {href}
   on:click={onClick}
   class={` flex h-[100px] items-center justify-center text-sm font-bold ${
-    isActive && 'bg-bg-black-active shadow-blue-border'
+    isActive && 'bg-cream text-teal shadow-blue-border'
```

Comment:
Didn't look quite right to me, but I think this is better. Changing the hex in "shadow-blue-border" to be the bright blue too.
```suggestion
    isActive && 'bg-cream text-bright-blue shadow-blue-border'
```